### DISCUSSION TITLE
How to hide notifications below a configured level?

### DISCUSSION THREAD

**Discussion by baodrate** [folke/snacks.nvim#82] [Q&A] [answered]
nvim-notify [hides notifications below a configured level](https://github.com/rcarriga/nvim-notify/blob/fbef5d32be8466dd76544a257d3f3dce20082a07/lua/notify/config/init.lua#L46-L47)[^1] but still shows them in the notification history[^2]

[^1]: set to `vim.log.levels.INFO` [by default](https://github.com/rcarriga/nvim-notify/blob/fbef5d32be8466dd76544a257d3f3dce20082a07/lua/notify/config/init.lua#L21)
[^2]: accessible in nvim-notify with `:Notifications` and in snacks.notifier with `:lua require("snacks.notifier").show_history()`

Is it possible to copy this behavior with snacks.notifier? Since the switch to a snacks.notifier in LazyVim, I'm seeing more debug-level log messages I didn't before. Some of them are from my config and can be disabled, but others are from plugins.

**Comment by folke** [comment, marked as answer]
Good point. I didn't have that since noice sits in between.
Will add.

**Reply by folke** [reply]
Has been added. Check the updated docs.
